### PR TITLE
Enhance rangeFromLineNumber behavior

### LINKED ISSUE #137: Make rangeFromLineNumber highlight the whole line if no column is provided
Occasionally it would still be desirable to highlight the entire rest of the line, even if a word can be determined (see https://github.com/AtomLinter/linter-pyflakes/pull/38#issuecomment-228419569).

Add a flag to `rangeFromLineNumber` making it behave as it did in the previous version.

### PR DESCRIPTION
**Pull request by steelbrain**
Fixes #137

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -28,21 +28,21 @@ export function rangeFromLineNumber(textEditor: TextEditor, line: ?number, colum
...
-  let colEnd = lineLength
+  const lineText = buffer.lineForRow(lineNumber)
+  let colEnd = lineText.length
   let colStart = columnGiven ? column : 0
-  const match = Helpers.getWordRegexp(textEditor, [lineNumber, colStart]).exec(lineText)
-  if (match) {
-    colEnd = colStart + match.index + match[0].length
-    if (!columnGiven) {
-      colStart = match.index
+  if (columnGiven && column > lineText.length) {
```
Comment (Arcanemagus) [inline src/index.js]: Any reason this check is behind the `columnGiven` flag? Shouldn't it be at the end of all figuring out of the end column?

Comment (steelbrain) [inline src/index.js]: The reason this is behind the `columnGiven` flag is that we don't need to validate this if we generate it ourselves from the line text

Comment (Arcanemagus) [inline src/index.js]: Assuming the generation works perfectly every time 😛.

Comment (steelbrain) [inline src/index.js]: Moved it to the end

**Pull request merged**